apps/web/components/booking/BookingListItem.tsx

```diff
@@ -134,7 +153,8 @@ function BookingListItem(booking: BookingItemProps) {
          cancel all remaining bookings or just that booking instance. */
       href: `/booking/${booking.uid}?cancel=true${
         isTabRecurring && isRecurring ? "&allRemainingBookings=true" : ""
-      }`,
+      }${booking.seatsReferences.length ? `&seatReferenceUid=${getSeatReferenceUid()}` : ""}
+      `,
```

Comment:
We pass the `seatsReferenceUId` as a URL param to the cancel booking page. With the uid of the booking and seat referenceId it would be hard for someone to guess the right combination